_tutorials/part-5-packaging-and-distributing-plugins.md

```diff
@@ -33,23 +56,46 @@ You can split anything out to anywhere you want. The only rule is that the
 things that the bot is searching for (basically, just the things that you're
 decorating with the decorators in `sopel.module`) need to be present in that
 `__init__.py`. So if you had a callable `eggs` in there, and you move it into
-`~/.sopel/modules/spam/callables.py`, you'll want to have
-`from .callables import eggs` (note that leading `.`, to say we want it from
-the `callables` module in the same directory). It's worth noting that you won't
-be able to import things from `__init__.py` in other files in the directory
-reliably, so it's probably best to have `__init__.py` do nothing but import the
-things it needs.
+`~/.sopel/modules/spam/callables.py`, you'll want to have `from .callables
```

Comment:
Do you think we can change to `./sopel/plugins` instead?

Reply:
🤦‍♂️ Why didn't I do this.